src/ctelnet.cpp

```diff
@@ -1573,11 +1573,6 @@ void cTelnet::_loadReplay()
 
         char* pB = &loadBuffer[0];
         loadedBytes = replayStream.readRawData(pB, amount);
-        qDebug("_loadReplay(): loaded: %i/%i bytes, wait for %1.3f seconds. (Single shot duration is: %1.3f seconds.)",
```

Comment:
This is useful if you are __debugging__ the ___replay code___ as if your test replay has long periods of time when nothing happens it prints out how long the gap to the next chunk's arrival taking into account the speed-up factor (I had a mod some years ago - that I never included -  that caused the on-toolbar time display to approximate to incrementing once per second - and correcting itself to the exact time when the chunk came in - because currently the time display only updates with each chunk...)